File: Go/fizzbuzz.go
package main

import (
        "fmt"
)

func main() {
        var inputA, inputB, inputC int
        fmt.Scanf("%d %d %d", &inputA, &inputB, &inputC)

        for i := 1; i <= inputC; i++ {
                if i%inputA == 0 && i%inputB == 0 {
                        fmt.Println("FizzBuzz")
                } else if i%inputB == 0 {
                        fmt.Println("Buzz")
                } else if i%inputA == 0 {
                        fmt.Println("Fizz")
                } else {
                        fmt.Println(i)
                }
        }
}

File: Problems/fizzbuzz.py
a, b, c = input().split()

for i in range(1, int(c)+1):
    a, b, c = int(a), int(b), int(c)
    if i % a == 0 and i % b == 0:
        print('FizzBuzz')
    elif i % b == 0:
        print('Buzz')
    elif i % a == 0:
        print('Fizz')
    else:
        print(i)
